src/content/LandingPage/LandingPage.js

```diff
@@ -1,7 +1,106 @@
 import React from 'react';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  Button,
+  Tabs,
+  Tab,
+} from 'carbon-components-react';
 
 const LandingPage = () => {
-  return <div>LANDING PAGE</div>;
+  const props = {
+    tabs: {
+      selected: 0,
+      triggerHref: '#',
+      role: 'navigation',
+    },
+    tab: {
+      href: '#',
+      role: 'presentation',
+      tabIndex: 0,
+    },
+  };
+
+  return (
+    <div className="bx--grid bx--grid--full-width landing-page">
+      <div className="bx--row landing-page__banner">
+        <div className="bx--row landing-page__banner">
+          <div className="bx--col-lg-16">
+            <Breadcrumb noTrailingSlash aria-label="Page navigation">
+              <BreadcrumbItem>
+                <a href="/">Getting started</a>
+              </BreadcrumbItem>
+            </Breadcrumb>
+            <h1 className="landing-page__heading">
+              Design &amp; build with Carbon
+            </h1>
+          </div>
+        </div>
+      </div>
+      <div className="bx--row landing-page__r2">
+        <div className="bx--row landing-page__r2">
```

Comment:
This also looks like you have an unnecessary, duplicated `<div>` here.

Reply:
Duplication removed!